### PR TITLE
LPS-159475 Add tests CurrentStepIsSavedWhenNavigatingPages and CurrentStepIsSavedWhenUserSignsInAndOut

### PR DESCRIPTION
**Pull request by susanchen3**
**Background**
Create automated tests for Walkthrough component.

**Test Plan**
Given walkthrough enabled site-level with multiple pages defined
When on step 2
And When navigate to previous step through url
Then popover mode is not present
When navigate back to next step through url
Then popover mode element is present on step 2

Given walkthrough enabled site-level with multiple pages defined
And Given on step 2 in popover mode
When user signs out
And When user signs in
Then popover mode is present on step 2

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-159475
https://issues.liferay.com/browse/LPS-159476

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
Will need to follow up and quarantine the RemoteAppsUpgrade test out of frontend test suite, tired of seeing those tests fail...

**Comment by john-co** [comment]
Just started reviewing :)

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughMultiplePages.testcase [hunk 1]
```diff
@@ -118,4 +118,74 @@ definition {
...
+
+		task ("And When user signs out") {
+			User.logoutPG();
+		}
+
+		task ("And When user signs in") {
+			User.loginPG(
+				password = "test",
+				userEmailAddress = "test@liferay.com");
+		}
```
[the remark this answers is not in the document]
Comment (susanchen3) [reply modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughMultiplePages.testcase:180]: theres no logout macro that doesnt use UI, but the login macro uses url manipulation

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**